invenio_records_rest/views.py

```diff
@@ -69,6 +68,12 @@ def create_blueprint(endpoints):
         for rule in create_url_rules(endpoint, **options):
             blueprint.add_url_rule(**rule)
 
+    # catch record validation errors
+    @blueprint.errorhandler(ValidationError)
+    def validation_error(error):
+        """Catch validation errors."""
+        return RESTValidationError().get_response()
```

Comment:
I tried to raise but it returns error: ` TypeError: exceptions must be old-style classes or derived from BaseException, not Response`